@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -126,6 +126,13 @@ export class TileGeometryCreator {
     private static m_colorMap: Map<string, THREE.Color> = new Map();
 
     private static m_instance: TileGeometryCreator;
+    // These are used for clipping the geometry so that the overlapping geometry is removed.
+    private clippingPlanes = [
```

Comment:
These will not work in globe

Reply:
see the comment below `// TODO: Add clipping for Spherical projection.`

Reply:
> Please split this PR in two, one for the clip planes and one for the fallback render order

The problem with splitting it is that if you have the render order reduced, the fallback has weird seams at the end, because the other tiles spill over. IMO this is a regression and shouldn't be split.